app/src/main/res/values/strings.xml

```diff
@@ -77,6 +77,9 @@
   <string name="fraction_error_invalid_format">Please enter a valid fraction (e.g., 5/3 or 1 2/3)</string>
   <string name="fraction_error_divide_by_zero">Please do not put 0 in the denominator</string>
   <string name="fraction_error_larger_than_seven_digits">None of the numbers of the fraction should be larger than 7 digits.</string>
+  <string name="number_error_starting_with_floating_point">Please begin your answer with a number (e.g.,”0” in 0.5)</string>
+  <string name="number_error_invalid_format">Please enter a valid number </string>
```

Comment:
there is an extra space after `...number`. Remove this.

Reply:
Done